### PR TITLE
cart detail page without headers and links, close #194

### LINKED ISSUE #194: cart page

### PR DESCRIPTION
**Pull request by schleichardt**

### CODE REVIEW

**Comment by schleichardt** [comment]
@lauraluiz please review

File: common/app/common/models/ProductVariantBean.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    private String sku;
+    private long quantity;
+    private String priceOld;
+    private String price;
+    private String total;
+    private String variantId;
+    private String productId;
+    private List<ProductAttributeBean> attributes;
+
+    public ProductVariantBean(final LineItem lineItem, final UserContext userContext, final ProductDataConfig productDataConfig) {
```
Comment (lauraluiz) [inline common/app/common/models/ProductVariantBean.java]: I will use this class as well for products once you merge this

File: common/app/common/templates/HandlebarsTranslationHelper.java [hunk 2]
```diff
@@ -14,15 +14,14 @@
 import java.util.*;
 
 import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.toList;
 
 final class HandlebarsTranslationHelper extends Base implements Helper<String> {
```
Comment (lauraluiz) [inline common/app/common/templates/HandlebarsTranslationHelper.java]: I know it's not about this request, but I would rename this class to CustomI18n, so it is giving a hint that the intention is overriding I18n helper from Hbs Java.

File: common/app/common/utils/PriceFormatterImpl.java [hunk 3]
```diff
@@ -13,7 +19,13 @@
...
-    public String format(MonetaryAmount price) {
-        return MonetaryFormats.getAmountFormat(locale).format(price);
+    public String format(MonetaryAmount monetaryAmount) {
+        final AmountFormatQuery amountFormatQuery = AmountFormatQueryBuilder.of(locale)
+                .set("pattern", "###0.00### ¤")
+                .set(CurrencyStyle.SYMBOL)
+                .build();
+        final MonetaryAmountFormat amountFormat = MonetaryFormats.getAmountFormat(amountFormatQuery);
+        return amountFormat.format(monetaryAmount)
+                .replace("EUR", "€");//yep, the money lib does not know the euro sign
```
Comment (lauraluiz) [inline common/app/common/utils/PriceFormatterImpl.java]: You can use the same I implemented in dmexco version, that one was implemented and tested:
https://github.com/sphereio/sphere-sunrise-dmexco/blob/master/common/app/common/utils/PriceFormatterImpl.java

Comment (lauraluiz) [inline common/app/common/utils/PriceFormatterImpl.java]: And the money lib does know the euro sign, might be that you are not using a european locale?

Comment (lauraluiz) [inline common/app/common/utils/PriceFormatterImpl.java]: I mean, with country included: https://github.com/sphereio/sphere-sunrise-dmexco/blob/master/common/test/common/utils/PriceFormatterTest.java

**Comment by lauraluiz** [comment]
Reviewed

**Pull request merged**